ci/base-image/Dockerfile

```diff
@@ -1 +1 @@
-FROM node:10.23.1-buster
+FROM node:12.18.3-buster
```

Comment:
After changing of the docker image, don't we need to update `.buildkite/pipeline.yaml`? [DEVELOPING.md](https://github.com/radicle-dev/radicle-contracts/blob/master/DEVELOPING.md#updating-cis-base-docker-image)

Reply:
:heavy_check_mark: